wiki/tfchain/tfchain_home.md

```diff
@@ -0,0 +1,62 @@
+
+## Farms
+
+A farm must be created before a Node can be booted. Every farms needs to have an unique name and is linked to the Twin that creates the farm. Once a farm is created, a unique ID is generated. This ID can be used to provide to the boot image of a Node.
+
+## Nodes
+
+When a node is booted for the first time, it registers itself on the chain and a unique identity is generated for this Node.
+
+## Node Contract
+
+A node contract is a contract between a user and a Node to deploy a certain workload. The contract is specified as following:
+
+```
+{
+    "contract_id": auto generated,
+    "node_id": unique id of the node,
+    "deployment_data": some additional deployment data
+    "deployment_hash": hash of the deployment definition signed by the user
+    "public_ips": number of public ips to attach to the deployment contract
+}
+```
+
+We don't save the raw workload definition on the chain but only a hash of the definition. After the contract is created, the user must send the raw deployment to the specified node in the contract. He can find where to send this data by looking up the Node's twin and contacting that twin over the planetary network.
+
+## Rent Contract
+
+A rent contract is also a contract between a user and a Node, but instead of being able to reserve a part of the node's capacity, the full capacity is rented. Once a rent contract is created on a Node by a user, only this user can deploy node contracts on this specific node. A discount of 50% is given if a the user wishes to rent the full capacity of a node by creating a rent contract. All node contracts deployed on a node where a user has a rent contract are free of use expect for the public ip's which can be added on a node contract.
+
+## Name Contract
+
+A name contract is a contract that specifies a unique name to be used on the grid's webgateways. Once a name contract is created, this name can be used as and entrypoint for an application on the grid.
+
+## Contract billing
+
+Every contract is billed every 1 hour on the chain, the amount that is due is deducted from the user's wallet every 24 hours or when the user cancels his contract. The total amount acrued in those 24 hours gets send to following destinations:
+
+- 10% goes to the threefold foundation
+- 5% goes to staking pool wallet (to be implemented in a later phase)
+- 50% goes to certified sales channel
+- 35% TFT gets burned
+
+See [pricing](tfchain_pricing) for more information on how the cost for a contract is calculated.
+
+## Contract locking
+
+To not overload the chain with transfer events and others we choose to lock the amount due for a contract every hour and after 24 hours unlock the amount and deduct it in 1 go. This lock is saved on a user's account, if the user has multiple contracts the locked amount will be stacked.
+
+## Contract grace period
```

Comment:
Please add the grace periods expected for testnet and devnet too

Reply:
Grace periods are the same across all network since it's configured in the runtime and we don't have seperate runtimes for different networks